zrml/prediction-markets/src/tests.rs

```diff
@@ -661,68 +653,24 @@ fn create_market_and_deploy_assets_is_identical_to_sequential_calls() {
             amount_base_asset.clone(),
             amounts.clone(),
             weights.clone(),
```

Comment:
We can remove all `.clone()` calls within the function parameters of `create_cpmm_market_and_deploy_assets`

Reply:
Thanks! 👍